recipes/czmq/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "czmq" %}
+{% set version = "4.0.2" %}
+{% set sha256 = "794f80af7392ec8d361ad69646fc20aaa284d23fef92951334009771a732c810" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/zeromq/czmq/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  skip: True  # [win]
+  script: ./autogen.sh && ./configure && make && make install
+
+requirements:
+  build:
+    - libtool
+    - automake
+    - autoconf
+    - pkg-config
+    - zeromq 4.2.1
+  run:
+    - zeromq 4.2.1
+
+test:
+
+about:
+  home: http://zeromq.org
+  license: MIT
+  summary: High-level C bindings for 0MQ
```

Comment:
please include the `license_file` and `dev_url` fields. 

https://github.com/zeromq/czmq